src/rules/objectLiteralShorthandRule.ts

```diff
@@ -37,12 +49,37 @@ export class Rule extends Lint.Rules.AbstractRule {
         description: "Enforces/disallows use of ES6 object literal shorthand.",
         hasFix: true,
         optionsDescription: Lint.Utils.dedent`
-        If the \'never\' option is provided, any shorthand object literal syntax will cause a failure.`,
+        If the \'never\' option is provided, any shorthand object literal syntax will cause a failure.
```

Comment:
Can you please mention at the beginning of this description that by default, with no options specified, that it enforces shorthand always?

Also, for clarity, I think we should support the following config: `{ "property": "always", "method": "never" }` (and vice versa). We should mention that "always" is assumed the default if the respective key is not specified in the config, but we should also allow users to be explicit about it so that it's easier to read tslint.json.